### PR TITLE
Add: clarifying example

### PR DESCRIPTION
**Pull request by benkoshy**
### Why?

* examples are helpful.
* I often get confused/forget. 
* Perhaps others may benefit?

### CODE REVIEW

File: docs/reference/targets.md [hunk 1]
```diff
@@ -121,4 +121,15 @@ with a matching name, the corresponding callback _will not_ be invoked again.
 
 ## Naming Conventions
 
-Always use camelCase to specify target names, since they map directly to properties on your controller.
+Always use camelCase to specify target names, since they map directly to properties on your controller:
+
+```erb
```
Comment (lb-) [inline docs/reference/targets.md:126]: maybe make this `html` as not everyone who uses stimulus uses ruby.

Also use HTML syntax

File: docs/reference/targets.md [hunk 2]
```diff
@@ -121,4 +121,15 @@ with a matching name, the corresponding callback _will not_ be invoked again.
...
+<span data-search-target="camelCase" span/> 
+<span data-search-target="do-not-do-this" span/>
```
Comment (marcoroth) [inline docs/reference/targets.md:128]:
```suggestion
<span data-search-target="camelCase" /> 
<span data-search-target="do-not-do-this" />
```

File: docs/reference/targets.md [hunk 3]
```diff
@@ -121,4 +121,15 @@ with a matching name, the corresponding callback _will not_ be invoked again.
...
+```
+
+```js
+export default class extends Controller {
+  static targets = ["camelCase" ]
```
Comment (marcoroth) [inline docs/reference/targets.md:133]:
```suggestion
  static targets = [ "camelCase" ]  
```

File: docs/reference/targets.md (hunk 1 above)
Comment (benkoshy) [reply docs/reference/targets.md:126]: Oops. Nice catch.

File: docs/reference/targets.md (hunk 2 above)
Comment (benkoshy) [reply docs/reference/targets.md:128]: will force push. nice catch.

**Review by marcoroth — APPROVED** [review]
Thank you @benkoshy!

**Pull request merged**